src/lib/sw_engine/tvgSwFill.cpp

```diff
@@ -144,13 +144,13 @@ bool _prepareRadial(SwFill* fill, const RadialGradient* radial, const Matrix* tr
         fill->radial.cx = tx;
         fill->radial.cy = ty;
 
-        auto sx = sqrt(pow(transform->e11, 2) + pow(transform->e21, 2));
-        auto sy = sqrt(pow(transform->e12, 2) + pow(transform->e22, 2));
+        auto sx = sqrtf(pow(transform->e11, 2) + pow(transform->e21, 2));
+        auto sy = sqrtf(pow(transform->e12, 2) + pow(transform->e22, 2));
 
         //FIXME; Scale + Rotation is not working properly
         radius *= sx;
 
-        if (abs(sx - sy) > DBL_EPSILON) {
+        if (fabsf(sx - sy) > FLT_EPSILON) {
```

Comment:
@hermet please notice, that I changed it back to a float precision
are you ok with that?

Reply:
no, sx - sy are doubles, so DBL_EPSION is correct.

Reply:
oh you changed all others.